pepper-apis/src/main/java/org/broadinstitute/ddp/content/RenderValueProvider.java

```diff
@@ -168,6 +169,13 @@ public String isGovernedParticipant(String ifTrueString, String ifFalseString) {
         return governedParticipant != null && governedParticipant ? ifTrueString : ifFalseString;
     }
 
+    /**
+     * Get GUID of a snapshotted address
+     */
+    public String addressGuid() {
+        return addressGuid;
+    }
```

Comment:
Public methods in `RenderValueProvider` are exposed as `$ddp.<method_name>()` method calls in templates. We don't need to expose this in templates right now. Let's remove this method.

Reply:
Done